modules/nf-core/cellbender/removebackground/main.nf

```diff
@@ -0,0 +1,49 @@
+process CELLBENDER_REMOVEBACKGROUND {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'oras://community.wave.seqera.io/library/cellbender:0.3.0--c4addb97ab2d83fe':
+        'community.wave.seqera.io/library/cellbender:0.3.0--41318a055fc3aacb' }"
+
+    input:
+    tuple val(meta), path(h5ad)
+
+    output:
+    tuple val(meta), path("${prefix}.h5")               , emit: h5
+    tuple val(meta), path("${prefix}_cell_barcodes.csv"), emit: barcodes
+    tuple val(meta), path("${prefix}_metrics.csv")      , emit: metrics
```

Comment:
What about the HTML and PDF reports? I think there's some value of publishing those to the output directory. 

Is the information which barcodes are "filtered" according to cellbender also contained elsewhere? Otherwise it might be worth additionally emitting the `${prefix}_filtered.h5` file.

Reply:
might be used for a multiqc module in the future: https://github.com/broadinstitute/CellBender/issues/257

Reply:
Added output emissions for all output files via the latest commits